lib/posthtml.js

```diff
@@ -10,12 +10,13 @@ function PostHTML(plugins) {
 }
 
 /**
- * Parse html to json tree
+ * Expose default parser and code generator
  *
  * @param {String} html htmltree
  * @returns {PostHTMLTree} json jsontree
  */
 PostHTML.parse = parser;
+PostHTML.generate = generator;
```

Comment:
I would also propose removing these here. Since `posthtml-parser` and `posthtml-render` are direct dependencies of `posthtml`, if anyone wants to access them, they can just require them.